src/extension.ts

```diff
@@ -69,6 +67,10 @@ export async function activate(
   // before considering TabNine ready to operate.
   void backgroundInit(context);
 
+  if (context.extensionMode !== vscode.ExtensionMode.Test) {
+    handlePluginInstalled(context);
+  }
```

Comment:
i think its more reasonable to move this code to inside `backgroundInit` - wdyt?

Reply:
I'm ok with that. but why? I see the uninstalled handler is also in this scope